app/Concerns/HasTags.php

```diff
@@ -33,7 +33,7 @@ public function withTags(): static
      */
     public function addTags(array $tags): static
     {
-        $this->tags = array_unique(array_merge($this->tags, $tags));
+        $this->tags = array_filter(array_unique(array_merge($this->tags, $tags)));
```

Comment:
Would a collection class help here? I've got one that I built a bit ago but have never used. I just made it public if you want to check it out: [myth-collections](https://github.com/lonnieezell/myth-collection)

Reply:
This library looks pretty cool, although I don't see an equivalent to the `array_unique` function (I could be wrong). The `array_filter` function is used here to remove empty values. It would be cool if myth-collections could make it happen without the need to write custom implementation in the `filter` method (but again, I could be wrong).

Right now I think we would produce much more code to achieve the same effect.

Reply:
You're probably right. And I am surprised I never put in a filter function. I'll have to remedy that at some point. It's too useful.